pkg/operation/botanist/component/etcd/etcd.go

```diff
@@ -663,7 +663,9 @@ func (e *etcd) computeContainerResources(existingSts *appsv1.StatefulSet) (*core
 				corev1.ResourceMemory: resource.MustParse("1G"),
 			},
 			Limits: corev1.ResourceList{
-				corev1.ResourceCPU:    resource.MustParse("2300m"),
+				// TODO This limit can't be removed currently because etcd-druid supplies a default of 100m
+				// Use a very high value instead and remove it later after etcd-druid is fixed
+				corev1.ResourceCPU:    resource.MustParse("10000"),
```

Comment:
When is the new etcd-druid version supposed to arrive? How hard it is to release and include it in the current g/g release, so that I will not have to spend even more effort on this PR? Can I perhaps help with that, instead of splitting?

Reply:
Well, it should have been released this week, but there are still relevant open PRs which need to be merged first (before the team conducts testing, I think). You can sync with @ashwani2k and @ishan16696 about it.

Reply:
There is a PR for the new etcd-druid release: https://github.com/gardener/gardener/pull/5693

Reply:
@rfranzke #5693 is now open (and mergeable), I assume this means we could close this discussion and unhold the PR?

Reply:
Yes, let's wait for this upgrade PR to be merged (soon/today) and then we can also merge your PR here.

Reply:
In fact, since https://github.com/gardener/etcd-druid/pull/312 has also been included in the new `etcd-druid` release, I could get rid of the TODOs for `Etcd` resource limits once it's merged. Do you think it will be merged soon?

Reply:
Yes, I think so. Please get rid of the TODOs :)

Reply:
/hold cancel